Automattic/wp-calypso
# Stats: Fix redirection of the invalid site for the post details page

Related to #70749

## Proposed Changes

* Remove the appended site fragment for redirection to the default Stats page.

## Testing Instructions



* Spin up this change with the Calypso Live link.
* Navigate to the Stats `Post Detail` page with an `invalid` site (`/stats/post/${post-id}/${invalid-site-slug}`).
* Ensure the redirection navigates to the main site's `Traffic` page (`/stats/day/${site-slug}`) or the `Overview` page (`/stats/day`) if you own multiple blogs.
* Ensure the URL is not appended with the wrong site query string.

## Pre-merge Checklist
